changelog.md

```diff
@@ -85,6 +85,11 @@ proc enumToString*(enums: openArray[enum]): string =
   (default value: true) that can be set to `false` for better Posix
   interoperability. (Bug #9619.)
 
+- The `sequtils.map` proc has been removed.
```

Comment:
Maybe specify that only the signature accepting a `var openArray` has been removed?

Reply:
+1 to a comment, sounds like one of the most frequently used procs in sequtils was removed. Needs more clarify what was actually removed